src/styles/app.scss

```diff
@@ -4,7 +4,7 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   @apply text-grey-900;
-  @apply bg-smoke-25;
+  @apply bg-white;
 }
 
 body.dark {
```

Comment:
dang my bad, thought i made this change already.

light mode is white ✅ , but i think dark mode should be `bg-slate-900` instead of 800.